.github/workflows/modular-arch-quality-gates.yml

```
@@ -0,0 +1,423 @@
+        with:
+          name: quality-gates-summary
+          path: quality-gates-summary.md
+
+      - name: Post Quality Gates Summary as PR Comment
+        if: github.event_name == 'pull_request'
+        uses: actions/github-script@v7
+        continue-on-error: true
+        with:
+          script: |
+            const fs = require('fs');
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)